### PR TITLE
Update index.md new sprig content

### PR DESCRIPTION
**Pull request by nilssprig**

### CODE REVIEW

File: src/connections/destinations/catalog/sprig-cloud/index.md [hunk 1]
```diff
@@ -8,16 +8,21 @@ versions:
...
     link: /docs/connections/destinations/catalog/sprig-web
 ---
-[Sprig (formerly UserLeap)](https://sprig.com/?&utm_source=segment_2021-10-20&utm_medium=int&utm_campaign=integration) is an in-context user research platform that makes it fast and effortless for product teams to learn from their actual customers in real time, through microsurveys, concept tests, and video questions.
+
+[Sprig](https://sprig.com/?&utm_source=segmentio&utm_medium=docs_actions&utm_campaign=integration) is an in-context user research platform that makes it fast and effortless for product teams to learn from their actual customers in real-time, through In-Product Surveys, Concept and Usability tests.
 
 Sprig maintains this destination. For any issues with the destination, consult [Sprig's documentation](https://docs.sprig.com/docs/segment) or contact [support@sprig.com](mailto:support@sprig.com).
 
-## Getting Started
+Segment has placed the Classic destination framework in maintenance mode. Sprig encourages all customers to move to the [Sprig Actions](https://segment.com/docs/connections/destinations/catalog/sprig-web/) destination framework when convenient.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/sprig-cloud/index.md:16]:
```suggestion
Segment has placed the Classic destination framework in maintenance mode. Sprig encourages all customers to move to the [Sprig Actions](/docs/connections/destinations/catalog/sprig-web/) destination framework when convenient. 
```

Please use relative URLs for links to other Segment Docs pages.

File: src/connections/destinations/catalog/sprig-cloud/index.md [hunk 2]
```diff
@@ -8,16 +8,21 @@ versions:
...
-2. Search for "Sprig Cloud" in the Destinations Catalog, and select the Sprig Cloud destination.
-3. Choose which Source should send data to the Sprig Cloud destination.
-4. Go to the [Sprig Connect page](https://app.sprig.com/connect), and find and copy the Segment **API key**. Use the Development key for a testing environment, and the Production key for your live environment.
-5. Enter the API Key that you copied in the Sprig Cloud destination settings in Segment.
+1. In the Segment web app, click **Catalog**, then click **Destinations**.
+2. Type Sprig in the **Filter Destinations** field.
+3. Click **Sprig**, then click **Configure Sprig**.
+4. Select an existing JavaScript website source to connect to Sprig and click **Next**.
+5. Type in a **Destination name**, Select **Classic**, and click **Save**.
+6. Type in the Environment ID and click Save Changes. You your Environment ID can be found in [Connect > JavaScript](https://app.sprig.com/connect). For for information, click [here](https://docs.sprig.com/docs/products-and-environments#environments).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/sprig-cloud/index.md:27]:
```suggestion
6. Type in the Environment ID and click Save Changes. You your Environment ID can be found in [Connect > JavaScript](https://app.sprig.com/connect){:target="_blank". For for information, click [here](https://docs.sprig.com/docs/products-and-environments#environments){:target="_blank"}.
```

File: src/connections/destinations/catalog/sprig-cloud/index.md [hunk 3]
```diff
@@ -8,16 +8,21 @@ versions:
   - name: Sprig (Actions)
     link: /docs/connections/destinations/catalog/sprig-web
 ---
-[Sprig (formerly UserLeap)](https://sprig.com/?&utm_source=segment_2021-10-20&utm_medium=int&utm_campaign=integration) is an in-context user research platform that makes it fast and effortless for product teams to learn from their actual customers in real time, through microsurveys, concept tests, and video questions.
+
+[Sprig](https://sprig.com/?&utm_source=segmentio&utm_medium=docs_actions&utm_campaign=integration) is an in-context user research platform that makes it fast and effortless for product teams to learn from their actual customers in real-time, through In-Product Surveys, Concept and Usability tests.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/sprig-cloud/index.md:12]:
```suggestion
[Sprig](https://sprig.com/?&utm_source=segmentio&utm_medium=docs_actions&utm_campaign=integration){:target="_blank"} is an in-context user research platform that makes it fast and effortless for product teams to learn from their actual customers in real-time, through In-Product Surveys, Concept and Usability tests.
```

For links away from Segment Docs, please open them in a new tab.

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/sprig-cloud/index.md [hunk 4]
```diff
@@ -8,16 +8,21 @@ versions:
...
 
 Sprig maintains this destination. For any issues with the destination, consult [Sprig's documentation](https://docs.sprig.com/docs/segment) or contact [support@sprig.com](mailto:support@sprig.com).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/sprig-cloud/index.md:14]:
```suggestion
Sprig maintains this destination. For any issues with the destination, consult [Sprig's documentation](https://docs.sprig.com/docs/segment){:target="_blank"} or contact [support@sprig.com](mailto:support@sprig.com).
```

File: src/connections/destinations/catalog/sprig-cloud/index.md [hunk 5]
```diff
@@ -8,16 +8,21 @@ versions:
...
+## Getting Started with Classic
 
 {% include content/connection-modes.md %}
 
-1. From the Destinations catalog page in the Segment App, click **Add Destination**.
-2. Search for "Sprig Cloud" in the Destinations Catalog, and select the Sprig Cloud destination.
-3. Choose which Source should send data to the Sprig Cloud destination.
-4. Go to the [Sprig Connect page](https://app.sprig.com/connect), and find and copy the Segment **API key**. Use the Development key for a testing environment, and the Production key for your live environment.
-5. Enter the API Key that you copied in the Sprig Cloud destination settings in Segment.
+1. In the Segment web app, click **Catalog**, then click **Destinations**.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/sprig-cloud/index.md:22]:
```suggestion
1. In the Segment web app, navigate to **Catalog > Destinations**.
```

File: src/connections/destinations/catalog/sprig-cloud/index.md [hunk 6]
```diff
@@ -8,16 +8,21 @@ versions:
...
+2. Type Sprig in the **Filter Destinations** field.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/sprig-cloud/index.md:23]:
```suggestion
2. Type *Sprig* in the **Filter Destinations** field.
```

File: src/connections/destinations/catalog/sprig-cloud/index.md [hunk 7]
```diff
@@ -8,16 +8,21 @@ versions:
...
+3. Click **Sprig**, then click **Configure Sprig**.
+4. Select an existing JavaScript website source to connect to Sprig and click **Next**.
+5. Type in a **Destination name**, Select **Classic**, and click **Save**.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/sprig-cloud/index.md:26]:
```suggestion
5. Enter a **Destination name**, select **Classic**, and click **Save**.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Review by stayseesong — APPROVED** [review]

**Pull request merged**